src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -372,6 +372,10 @@ public JabRefFrame frame() {
         return frame;
     }
 
+    public StateManager stateManager() {
+        return stateManager;
+    }
+
```

Comment:
This is something to be injected, not to be get. Inversion of dependency!

Reply:
Thanks for the explanation

Reply:
I passed the stateManager to the constructors instead of creating a new one

Reply:
Great!